### DISCUSSION TITLE
Azure blob storage parquet export

### DISCUSSION THREAD

**Discussion by Jbcot** [duckdb/duckdb#11354] [Ideas]
hello, is it planned to have a function to export parquet files to Azure blob storage gen1/gen2 as it already exists for S3 and GCS buckets ?(https://duckdb.org/docs/guides/import/s3_export) 

thank you in advance

regards

Jib

**Comment by quentingodeau** [comment]
Hi,
You can take a look at the [duckdb_azure repo](https://github.com/duckdb/duckdb_azure) the write operation are not yet done due to some issues with the `FileOpener` in the APIs but this has been fix today so it should be possible to implement now (I will rebase my branch and continue with what has been merge). 
Regards,
Quentin

**Reply by Jbcot** [reply]
Thanks a lot !
